ruddock/decorators.py

```diff
@@ -40,7 +40,7 @@ def wrapped_function():
 
       # We might need to change the form keys a bit
       form = flask.request.form
-      modified_form = {k.replace('-', '_'): v for k, v in form.items()}
+      modified_form = {k.replace('-', '_'): v for k, v in list(form.items())}
```

Comment:
List constructor isn't needed here right since form.items() is already an iterable